defender/connection.py

```diff
@@ -19,7 +19,15 @@ def get_redis_connection():
     if config.MOCK_REDIS:  # pragma: no cover
         return mocked_redis  # pragma: no cover
     else:  # pragma: no cover
-        redis_config = parse_redis_url(config.DEFENDER_REDIS_URL)
+        if config.DEFENDER_REDIS_URL:
+            redis_config = parse_redis_url(config.DEFENDER_REDIS_URL)
+        else:
+            redis_config = {
+                "DB": 0,
+                "PASSWORD": None,
+                "HOST": config.REDIS_HOST,
+                "PORT": 6379,
+            }
```

Comment:
If we add DEFENDER_REDIS_URL is our settings file then parse_redis_url() will return ("DB": 0,
        "PASSWORD": None,
        "HOST": "localhost",
        "PORT": 6379)
which means that host will be localhost instead of url from settings files. The only way this will work is to remove/not add DEFENDER_REDIS_URL from our settings files( I don't need to update all settings). If you want it I will merge it.